scripts/controllers/searchController.js

```diff
@@ -0,0 +1,13 @@
+(function(module) {
```

Comment:
Comparing the `resourceController.js`, `resourceView.js`, `searchController.js`, and `searchView.js` files; it appears that you've crossed some of the View / Controller borders between the two topics. For example, in resController you're not handling any of the view functionality, which leaves the controller light weight but in a good position to manage functionality as you scale.

One the other hand, the searchController is handling view functionality by hiding and showing elements. You should separate those view functions over to your searchView.js file and be sure you're controller module is handing a callback properly.